### PR TITLE
Feat/implement  UI detail page

### PR DESCRIPTION
**Pull request by tranchinh123**

### CODE REVIEW

File: bookshelter-app/src/pages/Detail/index.tsx [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+          </div>
+        </div>
+
+        <div className={styles.mainContent}>
+          <img
+            className={styles.imgBook}
+            src="https://i.ibb.co/ZSRxJHH/book-1.png"
+            alt=""
+          />
+          <span className={styles.description}>
```
Comment (huypham-agilityio) [inline bookshelter-app/src/pages/Detail/index.tsx]: why use span tag?

File: bookshelter-app/src/pages/Detail/index.tsx [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
+            <p className={styles.infoBookItem}>1996</p>
+            <p className={styles.infoBookItem}>Henry Mari</p>
+          </div>
+        </div>
+
+        <div className={styles.mainContent}>
+          <img
+            className={styles.imgBook}
+            src="https://i.ibb.co/ZSRxJHH/book-1.png"
+            alt=""
```
Comment (huypham-agilityio) [inline bookshelter-app/src/pages/Detail/index.tsx]: Please update `alt` for all `img` tag

File: bookshelter-app/src/pages/Detail/index.tsx [hunk 3]
```diff
@@ -1,8 +1,50 @@
...
+import Arrow2 from '../../components/Icons/Arrow2';
+import { Link } from 'react-router-dom';
 const DetailPage = () => {
   return (
-    <>
-      <h2>Detail Page</h2>
-    </>
+    <DefaultLayout>
+      <div className={styles.detailContentPage}>
+        <div className={styles.headerContent}>
```
Comment (huypham-agilityio) [inline bookshelter-app/src/pages/Detail/index.tsx]: should split this into component

File: bookshelter-app/src/pages/Detail/index.tsx [hunk 4]
```diff
@@ -1,8 +1,50 @@
...
+          <Link to="/" className={styles.backLink}>
+            <Arrow2 />
+            <span className={styles.backText}>Back</span>
+          </Link>
+          <p className={styles.nameBook}>Angels and Demons</p>
+        </div>
+
+        <div className={styles.info}>
```
Comment (huypham-agilityio) [inline bookshelter-app/src/pages/Detail/index.tsx]: same here

File: bookshelter-app/src/pages/Detail/index.tsx (hunk 1 above)
Comment (tranchinh123) [reply bookshelter-app/src/pages/Detail/index.tsx]: Fixed!

File: bookshelter-app/src/pages/Detail/index.tsx (hunk 2 above)
Comment (tranchinh123) [reply bookshelter-app/src/pages/Detail/index.tsx]: Fixed!

File: bookshelter-app/src/pages/Detail/index.tsx (hunk 3 above)
Comment (tranchinh123) [reply bookshelter-app/src/pages/Detail/index.tsx]: Fixed!

File: bookshelter-app/src/pages/Detail/index.tsx (hunk 4 above)
Comment (tranchinh123) [reply bookshelter-app/src/pages/Detail/index.tsx]: Fixed!

**Pull request merged**